src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        ColorSupplier supplier = new ColorSupplier();
+        String color = supplier.getRandomColor();
+        int num = new Random().nextInt(100);
```

Comment:
Magic number 100 should be extracted to a constant field with a clear name indicating its purpose, for example, `MAX_BALL_NUMBER`.